src/tools/wasm-split.cpp

```diff
@@ -453,6 +468,24 @@ uint64_t hashFile(const std::string& filename) {
   return uint64_t(digest);
 }
 
+void adjustTableSize(Module& wasm, const WasmSplitOptions& options) {
+  if (options.initialTableSize < 0) {
+    return;
+  }
+  if (!wasm.table.exists) {
+    Fatal() << "--initial-table used but there is no table";
+  }
+  if ((uint64_t)options.initialTableSize < wasm.table.initial) {
```

Comment:
Just pass options.initialTableSize as the second arg since not other options are used in this function?